tools/build-image/README.md

```diff
@@ -5,23 +5,19 @@ Grafana Alloy.
 
 There are two images:
 
-* `grafana/alloy-build-image:X.Y.Z` (for building Linux containers)
-* `grafana/alloy-build-image:X.Y.Z-windows` (for building Windows containers)
+* `grafana/alloy-build-image:vX.Y.Z` (for building Linux containers)
+* `grafana/alloy-build-image:vX.Y.Z-windows` (for building Windows containers)
```

Comment:
Is there a reason we're omitting a mention on the boringrypto image?

Reply:
I don't think so, I've added it. Thanks for raising!